simulaqron/tests/auto/quick/general/resources/sockets.cfg

```diff
@@ -5,6 +5,5 @@
 #
 # [name], [hostname], [port number]
 #
-Alice, localhost, 8801
-Bob, localhost, 8802
-Charlie, localhost, 8803
+Alice, localhost, 8001
+Bob, localhost, 8004
```

Comment:
Should this file not be deleted?

Reply:
No it is used to check that the old config files are still supported